honeybadgermpc/avss_value_processor.py

```diff
@@ -1,85 +1,104 @@
 import asyncio
 import logging
 from pickle import dumps, loads
+from collections import defaultdict
 from honeybadgermpc.protocols.commonsubset import run_common_subset
-from honeybadgermpc.exceptions import HoneyBadgerMPCError
-
-
-class NotEnoughAVSSValuesError(HoneyBadgerMPCError):
-    pass
+from honeybadgermpc.sequencer import Sequencer
 
 
 class AvssValueProcessor(object):
+    # How often to wait before checking if a value became available?
+    WAIT_PERIOD_IN_SECONDS = 0.1
```

Comment:
asynchronous purists will identify this as a potential red flag - timeout parameters will draw attention. It's OK, because we do not actually rely on these for safety OR for liveness. We should explicitly mark these as such, and just keep track of the proof obligation we need to explain

Reply:
I can think of a way to get rid of it and make the get by a _pure async_ method.

Reply:
Opened #218 to track this.